Source/Core/VideoCommon/ShaderCache.cpp

```diff
@@ -695,6 +696,35 @@ static GXPipelineUid ApplyDriverBugs(const GXPipelineUid& in)
     ps->ztest = EmulatedZ::EarlyWithZComplocHack;
   }
 
+  if (g_ActiveConfig.UseVSForLinePointExpand() &&
+      (out.rasterization_state.primitive.Value() == PrimitiveType::Points ||
+       out.rasterization_state.primitive.Value() == PrimitiveType::Lines))
```

Comment:
You *should* be able to just write `out.rasterization_state.primitive == PrimitiveType::Points` here (same applies for the line below and line 717).

Reply:
Indeed, I probably did that to get my autocomplete to suggest `PrimitiveType::` without actually remembering the enum name and forgot to remove it afterwards